src/dos/program_more.cpp

```diff
@@ -60,15 +61,79 @@ bool MORE::ParseCommandLine(MoreOutputFiles &output)
 	cmd->FillVector(params);
 
 	// Check if specified tabulation size
-	if (!params.empty()) {
-		const auto &param = params[0];
-		if ((starts_with("/t", param) || starts_with("/T", param)) &&
-		    (param.length() == 3) && (param.back() >= '1') &&
-		    (param.back() <= '9')) {
-			// FreeDOS extension - custom TAB size
-			output.SetTabSize(static_cast<uint8_t>(param.back() - '0'));
+	while (!params.empty()) {
+		auto is_option_extended = [](const std::string& param) {
+			return param == "/e" || param == "/E";
+		};
+		auto is_option_clear = [](const std::string& param) {
+			return param == "/c" || param == "/C";
+		};
+		auto is_option_expand_form_feed = [](const std::string& param) {
+			return param == "/p" || param == "/P";
+		};
+		auto is_option_squish = [](const std::string& param) {
+			return param == "/s" || param == "/S";
+		};
```

Comment:
Suggest the case-insensitive equality: `iequals(param, "/e")`
(in string_utils.h)

Reply:
OK, changed.